dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpBaseMessageImpl.java

```diff
@@ -2970,6 +2968,18 @@ else if (getServiceContext().getHttpConfig().onlySameSiteStar()) {
                 }
             }
 
+            if (cookie.getAttribute("samesite") != null && cookie.getAttribute("samesite").equals(HttpConfigConstants.SameSite.NONE.getName())) {
+                String userAgent = getServiceContext().getRequest().getHeader(HttpHeaderKeys.HDR_USER_AGENT).asString();
+                if (userAgent != null && SameSiteCookieUtils.isSameSiteNoneIncompatible(userAgent)) {
+                    //TODO: do we remove Secure, probably should be retained.
+                    cookie.setAttribute("samesite", null);
+                    // Partitioned should only be included when SameSite=None, so if it is incompatible Partitioned should be removed as well
+                    if (partitionedValue != null) {
+                        cookie.setAttribute("partitioned", null);
+                    }
+                }
```

Comment:
I would add a beta guard here as Paul mentioned. 

Add `import com.ibm.ws.kernel.productinfo.ProductInfo;` 
And then add a `ProductInfo.getBetaEdition()` check